Translation file required but not present

@msujew, it looks like we're still requiring the DE translations file, but it wasn't included in the PR. That produces an error on startup.

https://github.com/eclipse-theia/theia/blob/47dfd87a4143c66c4f8ed320bbec80ca0d6f67c7/packages/core/src/node/i18n/localization-contribution.ts#L103-L106


```
root ERROR Could not initialize contribution Error: Cannot find module '../../../i18n/nls.de.json'
Require stack:
- /home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/localization-contribution.js
- /home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/i18n-backend-module.js
- /home/evinfug/workspaces/theia-work/examples/electron/src-gen/backend/server.js
- /home/evinfug/workspaces/theia-work/examples/electron/src-gen/backend/main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at TheiaLocalizationContribution.registerLocalizations (/home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/localization-contribution.js:107:56)
    at /home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/localization-contribution.js:35:98
    at Array.map (<anonymous>)
    at LocalizationRegistry.initialize (/home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/localization-contribution.js:35:65)
    at LocalizationBackendContribution.initialize (/home/evinfug/workspaces/theia-work/packages/core/lib/node/i18n/localization-backend-contribution.js:33:41)
```

--- a/packages/core/src/node/i18n/i18n-backend-module.ts
+++ b/packages/core/src/node/i18n/i18n-backend-module.ts
@@ -18,7 +18,7 @@ import { ContainerModule } from 'inversify';
 import { localizationPath } from '../../common/i18n/localization';
 import { LocalizationProvider } from './localization-provider';
 import { ConnectionHandler, JsonRpcConnectionHandler, bindContributionProvider } from '../../common';
-import { LocalizationRegistry, LocalizationContribution, TheiaLocalizationContribution } from './localization-contribution';
+import { LocalizationRegistry, LocalizationContribution } from './localization-contribution';
 import { LocalizationBackendContribution } from './localization-backend-contribution';
 import { BackendApplicationContribution } from '../backend-application';
 
@@ -29,8 +29,6 @@ export default new ContainerModule(bind => {
     ).inSingletonScope();
     bind(LocalizationRegistry).toSelf().inSingletonScope();
     bindContributionProvider(bind, LocalizationContribution);
-    bind(TheiaLocalizationContribution).toSelf().inSingletonScope();
-    bind(LocalizationContribution).toService(TheiaLocalizationContribution);
     bind(LocalizationBackendContribution).toSelf().inSingletonScope();
     bind(BackendApplicationContribution).toService(LocalizationBackendContribution);
 });

--- a/packages/core/src/node/i18n/localization-contribution.ts
+++ b/packages/core/src/node/i18n/localization-contribution.ts
@@ -98,10 +98,3 @@ export class LocalizationRegistry {
         return undefined;
     }
 }
-
-@injectable()
-export class TheiaLocalizationContribution implements LocalizationContribution {
-    async registerLocalizations(registry: LocalizationRegistry): Promise<void> {
-        registry.registerLocalizationFromRequire('de', require('../../../i18n/nls.de.json'));
-    }
-}

